cli/src/stake.rs

```diff
@@ -326,52 +326,6 @@ impl StakeSubCommands for App<'_, '_> {
                 .arg(memo_arg())
                 .arg(compute_unit_price_arg()),
         )
-        .subcommand(
-            SubCommand::with_name("redelegate-stake")
```

Comment:
Yeah sure, no problem! I was thinking that it was fine in this case since it was never released, but it's nicer to get an informative error message than `error: Found argument '...' that wasn't expected`